db/migrate/20170831115230_create_certificate_model.rb

```diff
@@ -0,0 +1,20 @@
+class CreateCertificateModel < ActiveRecord::Migration[5.1]
+  def change
+    create_table :certificates do |t|
+      t.integer :hours
+      t.integer :minutes
+      t.date :duration_start
+      t.date :duration_end
+      t.text :institution
+      t.text :text_body
+      t.text :institution
+      t.string :function
+      t.jsonb :volunteer_contact
+      t.jsonb :assignment_kinds
+      t.references :volunteer, foreign_key: true
+      t.references :user, foreign_key: true
```

Comment:
`index: true` also usually makes sense with `#references`

Reply:
I see no difference in the resulting schema.rb with either 
```ruby
t.references :volunteer, foregin_key: true
t.references :volunteer, foreign_key:true, index: true
t.belongs_to :volunteer, foreign_key: true
```
They all give me the same output in schema.rb:
```ruby
create_table "certificates", force: :cascade do |t|
    t.integer "hours"
    t.integer "minutes"
    t.date "duration_start"
    t.date "duration_end"
    t.text "institution"
    t.text "text_body"
    t.string "function"
    t.jsonb "volunteer_contact"
    t.jsonb "assignment_kinds"
    t.bigint "volunteer_id"
    t.bigint "user_id"
    t.datetime "deleted_at"
    t.datetime "created_at", null: false
    t.datetime "updated_at", null: false
    t.index ["deleted_at"], name: "index_certificates_on_deleted_at"
    t.index ["user_id"], name: "index_certificates_on_user_id"
    t.index ["volunteer_id"], name: "index_certificates_on_volunteer_id"
  end
```

They're all resulting in the same, with an index.
And is it even possible, to create a foreign key in postgres, that doesn't have an index? So far I thought that this is not possible....